docs/user_guide/build_intel_boot_bin.rst

```diff
@@ -0,0 +1,850 @@
+     - 1
+   * - MSEL4
+     - 0
+   * - CODEC_SEL
+     - 0
+
+And set JP2 to 2.5V or 1.8V
+
+Configuring the SD Card
+```````````````````````````````````````````````````````````````````````````````
+
+Below is the commands to create the preloader and bootloader partition using
+the Kuiper Linux image as a starting point.
+Please check every command before running, especially configuring target
+device mountpoints accordingly
+(here as ``/dev/sdz`` with partition 1 mounted at ``/media/BOOT/``).
+
+Flash the SD Card with the Kuiper Linux image:
+
+.. shell::
+
+   $time sudo dd if=./2023-12-13-ADI-Kuiper-full.img of=/dev/sdz status=progress bs=4194304
+    2952+0 records in
+    2952+0 records out
+    12381585408 bytes (12 GB, 12 GiB) copied, 838.353 s, 14.8 MB/s
+
+    real	14m7.938s
+    user	0m0.006s
+    sys	0m0.009s
+   $sync
+
+Mount the /BOOT partition:
+
+.. shell::
+   :no-path:
+
+   $lsblk
+    NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+    sdz           8:48   1  29.1G  0 disk
+    ├─sdz1        8:49   1     2G  0 part
+    ├─sdz2        8:50   1  27.1G  0 part
+    └─sdz3        8:51   1     8M  0 part
+
+   $mkdir -p /media/BOOT/
+   $sudo mount /dev/sdz1 /media/BOOT/
+   $lsblk
+    NAME        MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+    sdz           8:48   1  29.1G  0 disk
+    ├─sdz1        8:49   1     2G  0 part /media/BOOT
+    ├─sdz2        8:50   1  27.1G  0 part
+    └─sdz3        8:51   1     8M  0 part
+
+Copy the built files to the /BOOT partition:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc ; pwd
+    ~/hdl/projects/arradio/c5soc
+   $cp ./software/bootloader/u-boot-socfpga/u-boot.scr /media/BOOT/
+   $cp soc_system.rbf /media/BOOT/
+   $mkdir -p /media/BOOT/extlinux
+   $cp ./software/bootloader/u-boot-socfpga/extlinux/extlinux.conf /media/BOOT/extlinux/
+   ~
+   $cp ~/linux/arch/arm/boot/dts/socfpga_cyclone5_sockit_arradio.dtb /media/BOOT/socfpga.dtb
+   $cp ~/linux/arch/arm/boot/zImage /media/BOOT/
+
+Unmount the /BOOT partition:
+
+.. shell::
+   :no-path:
+
+   $sudo umount /dev/sdz1
+   $lsblk
+    NAME        MAJ:MIN RM  SIZE  RO TYPE MOUNTPOINT
+    sdz           8:48  1   29.1G  0 disk
+    ├─sdz1        8:49  1      2G  0 part
+    ├─sdz2        8:50  1   27.1G  0 part
+    └─sdz3        8:51  1      8M  0 part
+
+Flash the preloader boot partition:
+
+.. shell::
+   :no-path:
+
+   $cd ~/hdl/projects/arradio/c5soc/software/bootloader/u-boot-socfpga ; pwd
+    ~/hdl/projects/arradio/c5soc/software/bootloader/u-boot-socfpga
+   $sudo dd if=/dev/zero of=/dev/sdz3 oflag=sync status=progress \
+   $    bs=$(sudo blockdev --getsize64 /dev/sdz3) count=1
+    1+0 records in
+    1+0 records out
+    8388608 bytes (8.4 MB, 8.0 MiB) copied, 0.359183 s, 23.4 MB/s
+   $sudo dd if=./u-boot-with-spl.sfp of=/dev/sdz3
+    1697+1 records in
+    1697+1 records out
+    868996 bytes (869 kB, 849 KiB) copied, 0.21262 s, 4.1 MB/s
+
+.. _build_intel_boot_bin de10nano:
+
+CN0540/DE10Nano
```

Comment:
Same comments as for the previous example

Reply:
Still present. All the comments I had for the previous sections, they apply to this one as well, since it's the same text used for each example